R/utils_accessibility.R

```diff
@@ -126,53 +126,65 @@ nhs_card_tabstop <- function(header, tabindex = 0, ...) {
 }
 
 
-
-
-
-
-
 #' accessible_radio_button
 #'
-#' @param ... radio content
+#' @inheritParams shiny::radioButtons
 #'
-#' @noRd
+#' @return HTML
+#' @export
```

Comment:
If the function is meant to be used outside the package itself (i.e. not an _internal_ function), you have to include `@export`. This will add it to the `NAMESPACE`.